pipeline/prometheus.yml

```diff
@@ -49,19 +49,22 @@ resource_types:
   type: docker-image
   source:
     repository: ljfranklin/terraform-resource
-    tag: 0.11.14
+    tag: 0.12.6
 
 resources:
 
 - name: tfstate
   type: terraform
   source:
-    storage:
+    env_name: ((env))
+    backend_type: s3
+    backend_config:
       bucket: ((terraform_storage_bucket_name))
-      bucket_path: ((terraform_storage_bucket_path))
```

Comment:
terraform_storage_bucket_path param should be removed from pipeline variable and the readme too

Reply:
Thx, missed that.